lib/modules/dosomething/dosomething_api/resources/signup_resource.inc

```diff
@@ -99,7 +107,12 @@ function _signup_resource_access() {
  * @param boolean $competition
  * @return array
  */
-function _signup_resource_index($user, $campaigns, $count, $page, $competition) {
+function _signup_resource_index($user, $users, $campaigns, $count, $page, $competition) {
+
+  if (isset($users)) {
+    $user = $users;
```

Comment:
I think my recommendation would be for it to be the other way around, so eventually we could deprecate the singular version and stay with the plural form.